contracts/JBTiered721Delegate.sol

```diff
@@ -762,4 +763,26 @@ contract JBTiered721Delegate is IJBTiered721Delegate, JB721Delegate, Ownable {
     _tokenId;
     _tier;
   }
+
+  /**
+    @notice Return royalty info
+    @param _tokenId Token Id
+    @param _salePrice Sales price
+    @return receiver royalty receiver address
+    @return royaltyAmount Royalty Amount
+  */
+  function royaltyInfo(uint256 _tokenId, uint256 _salePrice) external view override returns (address receiver, uint256 royaltyAmount) {
+      _tokenId; //avoid compiler warning
+      // Get the project's current funding cycle metadata.
+      (, JBFundingCycleMetadata memory _metadata) = IJBController(directory.controllerOf(projectId)).currentFundingCycleOf(projectId);
+      // 4 specifies `useRoyalty` is true
+      if (_metadata.metadata == 4) {
```

Comment:
Q: Won't this clash with some of the other (existing) metadata settings?

Reply:
I checked the v3 repo didn't find any clashes so think we should be good